clang/test/Sema/warn-lifetime-analysis-nocfg.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsyntax-only -Wdangling -Wdangling-field -Wreturn-stack-address -verify %s
+// RUN: %clang_cc1 --std=c++20 -fsyntax-only -Wdangling -Wdangling-field -Wreturn-stack-address -verify %s
```

Comment:
Should we limit all the tests here to C++20?

Reply:
I think this should be fine. The analysis being tested is not specific to C++20. The use of concepts in the tests now requires C++20.